From c0f4276b3761ee1cb638e877e090728dbc74b4fa Mon Sep 17 00:00:00 2001
From: Kylie McClain <somasis@exherbo.org>
Date: Thu, 21 Jul 2016 00:10:45 -0400
Subject: examples: nfct-daemon: Fix test building on musl libc

On musl libc, sys/select.h is where FD_* definitions are located.

Signed-off-by: Kylie McClain <somasis@exherbo.org>
Signed-off-by: Pablo Neira Ayuso <pablo@netfilter.org>
---
 examples/netfilter/nfct-daemon.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/netfilter/nfct-daemon.c b/examples/netfilter/nfct-daemon.c
index a97c2ec..d223ac2 100644
--- a/examples/netfilter/nfct-daemon.c
+++ b/examples/netfilter/nfct-daemon.c
@@ -13,6 +13,7 @@
 #include <netinet/in.h>
 #include <inttypes.h>
 #include <errno.h>
+#include <sys/select.h>
 
 #include <libmnl/libmnl.h>
 #include <linux/netlink.h>
-- 
cgit v1.2.1

